xml/admin_rbd.xml

```diff
@@ -155,135 +155,180 @@
    After you create a &rbd;, you can use it like any other disk device: format
    it, mount it to be able to exchange files, and unmount it when done.
   </para>
-
-  <procedure>
-   <step>
-    <para>
-     Make sure your &ceph; cluster includes a pool with the disk image you want
-     to map. Assume the pool is called <literal>mypool</literal> and the image
-     is <literal>myimage</literal>.
-    </para>
-<screen>&prompt.cephuser;rbd list mypool</screen>
-   </step>
-   <step>
+  <para>
+   The <command>rbd</command> command defaults to accessing the cluster using
+   the &ceph; <literal>admin</literal> user account. This account has full
```

Comment:
i believe there is no such account 'admin', we use 'cephadm' with an entity, it is used in prompts as well
```suggestion
   the &ceph; administrator user account &cephuser;. This account has full
```

Reply:
Why? 'admin' looks perfectly well to me. Usually (always?) you can find this account in a ceph cluster and it is used by default in CLI if any other user is specified via options.